identities/models.py

```diff
@@ -44,3 +44,11 @@ class Identity(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+
+class OptOut(models.Model):
+    """An opt_out"""
+    identity = models.ForeignKey(Identity)
```

Comment:
so then I should add a nullable address type and address pairing here as well? Just checking that I'm understanding correctly